llpc/patch/llpcPatchIntrinsicSimplify.cpp

```diff
@@ -87,7 +88,9 @@ void PatchIntrinsicSimplify::getAnalysisUsage(
 bool PatchIntrinsicSimplify::runOnFunction(
     Function& func) // [in,out] LLVM function to be run on.
 {
+    SmallVector<IntrinsicInst*, 32> workList;
```

Comment:
workList -> candidateCalls (or something equivalent)
I often avoid using worklist because it is too general to express what it is especially in large code piece. Readers will lose context.

Reply:
No problem, I have renamed to candidateCalls, I agree with the name.